ports/itpp/vcpkg.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "itpp",
+  "version-semver": "4.3.1",
+  "port-version": 6,
+  "description": "IT++ is a C++ library of mathematical, signal processing and communication classes and functions. Its main use is in simulation of communication systems and for performing research in the area of communications.",
+  "homepage": "http://itpp.sourceforge.net"
```

```suggestion
  "homepage": "http://itpp.sourceforge.net",
  "dependencies": [
    "fft"
  ]
```

I think this is necessary, right?

---

@strega-nil, we don't have fft library in vcpkg, so I haven't add it to the dependency.